tests/compute/credentials.go

```diff
@@ -117,6 +117,40 @@ var _ = SIGDescribe("Guest Access Credentials", func() {
 		),
 	)
 
+	It("[test_id:TODO] should remove authorized_keys file if secret is empty", func() {
+		vmi := libvmifact.NewFedora(withSSHPK(pubKeySecretID, v1.SSHPublicKeyAccessCredentialPropagationMethod{
+			QemuGuestAgent: &v1.QemuGuestAgentSSHPublicKeyAccessCredentialPropagation{
+				Users: []string{"fedora"},
+			},
+		}))
+
+		By("Creating a secret with empty file")
+		createNewSecret(testsuite.GetTestNamespace(vmi), pubKeySecretID, libsecret.DataBytes{
+			"my-empty-key": []byte{},
+		})
+
+		vmi = libvmops.RunVMIAndExpectLaunch(vmi, fedoraRunningTimeout)
+
+		By("Waiting for agent to connect")
+		Eventually(matcher.ThisVMI(vmi), guestAgentConnectTimeout, 2*time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceAgentConnected))
```

Comment:
Can we skip this assertion? Isn't it covered already by the assertion below?

Reply:
done